### ISSUE TITLE
[restrict-plus-operands] incorrect works with imported const numbers 

### ISSUE THREAD

**Issue by cevek** [typescript-eslint/typescript-eslint#1877]
**Repro**

```JSON
{
  "rules": {
    "@typescript-eslint/restrict-plus-operands": "error"
  }
}
```

```TS
// x.ts
export const x = 1
// b.ts
import {x} from './x';
const y = x + 1
```

**Expected Result**
no error

**Actual Result**
Operands of '+' operation must either be both strings or both numbers.(@typescript-eslint/restrict-plus-operands)

**Additional Info**

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `2.24.0` |
| `@typescript-eslint/parser`        | `2.24.0` |
| `TypeScript`                       | `3.8.3` |
| `ESLint`                           | `6.8.0` |
| `node`                             | `12` |
| `npm`                              | `6` |

**Comment by bradzacher** [comment]
I'm unable to repro this against master.
Could you please provide a repro repo?

**Comment by cevek** [comment]
Sorry, it is reproducible only my project. So I close this issue, and If I can to create repro I reopen it
